OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -182,13 +173,25 @@ Action InitDisplayPanel(Widget panel)
 			var cursorDoubleIsChecked = cursorDoubleCheckbox.IsChecked;
 			cursorDoubleCheckbox.IsChecked = () => !cursorDoubleCheckbox.IsDisabled() && cursorDoubleIsChecked();
 
-			panel.Get("WINDOW_RESOLUTION").IsVisible = () => ds.Mode == WindowMode.Windowed;
+			panel.Get("WINDOW_RESOLUTION").IsVisible = () => ds.Mode != WindowMode.NativeFullscreen;
```

Comment:
What? Definitely no, fullscreen (or pseudo-notsure of a sudden which I used) resolution change can be done via editing the settings.yaml on bleed. It makes sense as a form of full zooming in.